test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/ButtonSample.cs

```diff
@@ -28,186 +28,259 @@ public class ButtonSample : IExample
-            //imageView1.Size2D = new Size2D(48, 48);
-            //imageView1.Position2D = new Position2D(200, 200);
-            //imageView1.ResourceUrl = CommonResource.GetFHResourcePath() + "9. Controller/controller_btn_check.png";
-            //window.Add(imageView1);
-
-            //ImageView imageView2 = new ImageView();
-            //imageView2.Size2D = new Size2D(48, 48);
-            //imageView2.Position2D = new Position2D(500, 200);
-            //imageView2.ResourceUrl = CommonResource.GetFHResourcePath() + "9. Controller/[Controller] App Primary Color/controller_btn_check_on_24c447.png";
-            //window.Add(imageView2);
-
             root = new View()
             {
-                Size2D = new Size2D(1920, 1080),
+                Size = new Size(1920, 1080),
+                BackgroundColor = new Color(0.7f, 0.9f, 0.8f, 1.0f),
+                Layout = new LinearLayout()
+                {
+                    LinearAlignment = LinearLayout.Alignment.Center,
+                    LinearOrientation = LinearLayout.Orientation.Horizontal,
+                    CellPadding = new Size(50, 50),
+                }
             };
             window.Add(root);
             window.KeyEvent += Window_KeyEvent;
 
-            textButton = new Button();
-            textButton.Style.BackgroundImage = CommonResource.GetTVResourcePath() + "component/c_buttonbasic/c_basic_button_white_bg_normal_9patch.png";
-            textButton.Style.BackgroundImageBorder = new Rectangle(4, 4, 5, 5);
-            textButton.Size2D = new Size2D(300, 80);
-            textButton.Position2D = new Position2D(100, 100);
-	    textButton.Style.Text.TextShadow = new TextShadow(Color.Blue, new Vector2(2.0f, 2.0f), 5.0f);
-            textButton.Style.Text.Text = "Button";
-            root.Add(textButton);
+            parent1 = new View()
+            {
+                Size = new Size(300, 900),
+                Layout = new LinearLayout()
+                {
+                    LinearAlignment = LinearLayout.Alignment.Top,
+                    LinearOrientation = LinearLayout.Orientation.Vertical,
+                    CellPadding = new Size(50, 50),
+                }
+            };
+
+            parent2 = new View()
+            {
+                Size = new Size(300, 900),
+                Layout = new LinearLayout()
+                {
+                    LinearAlignment = LinearLayout.Alignment.Top,
+                    LinearOrientation = LinearLayout.Orientation.Vertical,
+                    CellPadding = new Size(50, 50),
+                }
+            };
 
+            // Only show a text button.
+            textButton = new Button();
+            var textStyle = textButton.Style;
+            textStyle.BackgroundColor = new Selector<Color>();
+            textButton.ApplyStyle(textStyle);
+            textButton.BackgroundImage = CommonResource.GetTVResourcePath() + "component/c_buttonbasic/c_basic_button_white_bg_normal_9patch.png";
```

Comment:
Can you remove the code getting Style and applying it after remove color selector?
I fixed related bug in #1630
```
textButton = new Button();
var textStyle = textButton.Style;
textStyle.BackgroundColor = new Selector<Color>();
textButton.ApplyStyle(textStyle);
textButton.BackgroundImage = ...
```

==>

```
textButton = new Button();
textButton.BackgroundImage = ...
```

Reply:
Removed